src/transaction.rs

```diff
@@ -359,6 +365,26 @@ pub struct MessageToL1 {
 )]
 pub struct EthAddress(pub H160);
 
+/// We could enable scale derivation with "codec" feature in "primitive-types" crate instead
```

Comment:
Please do. 
In Cargo.toml:
` parity-scale-codec = ["dep:parity-scale-codec", "primitive-types/codec"]`